### DISCUSSION TITLE
Error: Could not find authentication sk for pk

### DISCUSSION THREAD

**Discussion by temicus** [Chia-Network/chia-blockchain#8510] [Support]
**Describe the bug**
Getting every few minutes the Error:
Could not find authentication sk for pk: xxx...
I have this problem already since at least one month. 
Changing pools or solo modes bring nothing. Reinstalling or changing client version also doesn't help.
I had 2 wallets/keys on this machine before and removed in GUI one of them. Maybe its the result.

**To Reproduce**

1. Open the log file
2. check for errors

**Desktop**
win 10 pro, chia client 1.2.6.

**Comment by emlowe** [comment]
This means that many of your plots were plotted using the farmer key that you no longer have on that machine - if you don't have that key anymore, those plots cannot be farmed.

**Reply by digitalspaceport** [reply]
@emlowe is there a method to track down which plots are generating the "farmer chia.farmer.farmer         : ERROR    Could not find authentication sk for pk:"

**Reply by emlowe** [reply]
Actually, I was wrong about this error message. There are two similar ones:

`Could not find authentication sk for pk`: This is generated by the code when enumerating your pool_list in config.yaml - it can be cause if you at one point created some PlotNFTs for pooling, but then don't have that key anymore on this system (maybe you copied the config.yaml to a different system), or related issues. You can look in config.yaml for the matching pk printed on the log line and delete that entry from the pool_list

There is also `Plot xys has a farmer public key that is not in the farmer's pk list` and I had confused them

**Reply by digitalspaceport** [reply]
Oh that is awesome actually. Yup confirmed those keys for old test wallets did exist and removed them and the error is no more on restart.

**Comment by temicus** [comment]
@emlowe actually not.. all plots are definitely recognized :)
I have removed chia client completely, reinstalled it and synced and the error is gone.

**Reply by digitalspaceport** [reply]
@temicus did you delete the .chia folder after uninstalling? Could you outline the steps you did so I can follow you on this? I know I had some legacy keys installed on here, and I copied over the /config /wallet and /db directory on the new install, but did delete the rest of the chia folder (so I would not have issues with POOLED plots functionality.) 

Interestingly I am getting exactly what I should be from the pools for payout, but I am wondering if I may be handicapped in actually winning a block.

**Comment by BrandtH22** [comment]
@temicus , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
